terraform/application/config/qa.tfvars.json

```diff
@@ -1,5 +1,7 @@
 {
     "cluster": "test",
     "namespace": "bat-qa",
-    "enable_postgres_ssl": true
-}
+    "environment": "qa",
+    "deploy_azure_backing_services": false,
```

Comment:
Is this for testing?

Reply:
The looks like this on the branch:

    "cluster": "test",
    "namespace": "bat-qa",
    "environment": "qa",
    "deploy_azure_backing_services": false,
    "enable_postgres_ssl": false
  }

Should deploy_azure_backing_services and/or enable_postgres_ssl be set to true ?